eng/tools/typespec-validation/test/sdk-tspconfig-validation.test.ts

```diff
@@ -184,6 +184,27 @@ const tsManagementGenerateMetadataTestCases = createEmitterOptionTestCases(
   [new TspConfigTsMgmtModularGenerateMetadataTrueSubRule()],
 );
 
+const newTsManagementGenerateMetadataTestCases = createEmitterOptionTestCases(
```

Comment:
what is the case for?

Reply:
Forget to change line 190 to new generate-metadata